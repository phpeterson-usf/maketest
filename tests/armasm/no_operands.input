-s tests/armasm/no_operands.s
