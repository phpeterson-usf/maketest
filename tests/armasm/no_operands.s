@blah blah blah
main:

@    str r0, [sp, #4]
@    add r0, r1, r2
    mov r0, #0 @important stuff
foo:
    b 
nope:
    bx 
