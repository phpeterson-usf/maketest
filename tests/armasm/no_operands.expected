Error: Unexpected newline after 'b'. Expected operands
